modules/persistent-storage-csi-migration-vsphere-to-4-13.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-vsphere-to-4-13_{context}"]
+= Updating from {product-title} 4.12 to 4.13
+
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.12 to 4.13, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic CSI migration for vSphere only occurs if you opt in.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} update by running the following command to perform an administrator acknowledgment:
+
+[source, cli]
+----
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-126-vsphere-migration-in-4.14":"true"}}' --type=merge
+----
+
```

Comment:
I would like us to incorporate same wording as - https://github.com/openshift/openshift-docs/pull/65546#discussion_r1370294809 

because tbh, we want folks to upgrade to 4.13 and not discourage it.

Reply:
Agreed maybe rephrase the important square below to make it more positive.

Reply:
Done.